proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -111,8 +111,8 @@ message GetNodeInfoRequest {}
 
 // GetNodeInfoResponse is the response type for the Query/GetNodeInfo RPC method.
 message GetNodeInfoResponse {
-  .tendermint.p2p.DefaultNodeInfo default_node_info   = 1;
-  VersionInfo                     application_version = 2;
+  .tendermint.p2p.NodeInfo node_info           = 1;
```

Comment:
This is a proto-breaking change.

Maybe it's time to bump this package to v1

Reply:
I might be wrong here, but I believe this is just a naming change - the wire format should still be the same.

Reply:
Possibly. However, we're not allowing field name changes either anymore (due to too many headaches for client consumers). See [adr-044](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-044-protobuf-updates-guidelines.md#4-fields-must-not-be-renamed).

Tangent to this: are Tendermint proto packages not versioned?

Reply:
Yeah okay maybe we can bump this to `v1beta1`.

> Tangent to this: are Tendermint proto packages not versioned?

Yup, although that could change with Soft Upgrades work.

Reply:
The Tendermint proto packages are not versioned. I don't think we have any plans to do so. Ideally if the SDK wants to enforce a stronger API guarantee, it should use its own proto shim definitions.